crates/lang/src/lib.rs

```diff
@@ -0,0 +1,35 @@
+// ------ Lang ------
+
+use std::borrow::Cow;
+use std::fmt;
+
```

Comment:
Please add the line
```
@TODO: Add remaining langs.
```
so we can merge the PR and can continue in the future and not forget about it.